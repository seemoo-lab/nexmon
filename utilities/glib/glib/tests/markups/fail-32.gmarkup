<foo>&#x0;</foo>
