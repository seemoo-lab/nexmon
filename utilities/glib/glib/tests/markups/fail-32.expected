ELEMENT 'foo'
ERROR Error on line 1: Character reference '0' does not encode a permitted character
